examples/tools/bevymark.rs

```diff
@@ -72,67 +75,74 @@ fn scheduled_spawner(
             scheduled.per_wave,
             bird_texture.0.clone_weak(),
         );
-        counter.color = Color::rgb_linear(random(), random(), random());
+
+        let mut random = thread_rng();
+        counter.color = Color::rgb_linear(random.gen(), random.gen(), random.gen());
         scheduled.wave -= 1;
     }
 }
 
 struct BirdTexture(Handle<Image>);
 
+#[derive(Component)]
+struct StatsText;
+
 fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
     let texture = asset_server.load("branding/icon.png");
 
     commands.spawn_bundle(OrthographicCameraBundle::new_2d());
     commands.spawn_bundle(UiCameraBundle::default());
-    commands.spawn_bundle(TextBundle {
-        text: Text {
-            sections: vec![
-                TextSection {
-                    value: "Bird Count: ".to_string(),
-                    style: TextStyle {
-                        font: asset_server.load("fonts/FiraSans-Bold.ttf"),
-                        font_size: 40.0,
-                        color: Color::rgb(0.0, 1.0, 0.0),
+    commands
+        .spawn_bundle(TextBundle {
```

Comment:
I'm confused by this new newline; `rustfmt` should be deciding which one is correct

Reply:
Again, rustfmt did that (I added `.insert(StatsText)`).

Reply:
This comment was more based on an initial pass before CI passed; this change does make it harder to review, but that seems unavoidable.

Reply:
Ah, gotya. You can review the diff without whitespace if that helps: https://github.com/bevyengine/bevy/pull/3765/files?diff=unified&w=1